drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/rightpanel/TestToolsViewImpl.html

```diff
@@ -18,12 +18,16 @@
 <!-- TEST TOOLS -->
```

Comment:
It should be nice to move all `style` values to CSS even here

Reply:
@danielezonca Then, you now live in a nicer world :)